CHANGELOG.md

```diff
@@ -4,6 +4,7 @@ Mapbox welcomes participation and contributions from everyone.
 
 # main
 ## Features ✨ and improvements 🏁
+* Bump Mapbox Android base library to v0.8.0, to support incremental annotation processing. ([#1323](https://github.com/mapbox/mapbox-maps-android/pull/1323))
```

Comment:
perhaps we should divide in 2 - one entry for `Support incremental annotation processing` here and bump version to another block below

Reply:
Fixed